src/components/editMeasure/testCases/components/editTestCase/qiCore/LeftPanel/ElementsTab/builder/element/TypeEditor.tsx

```diff
@@ -162,8 +162,10 @@ const TypeEditor = ({
             structureDefinition={structureDefinition}
             fieldRequired={required}
             label={label}
-            onChange={onChange}
+            helperText={formikErrorHandler(label)}
+            error={getNestedProperty(formik.errors, label)}
             value={value}
+            {...formik.getFieldProps(label)}
```

Comment:
Nothing major, I think we can remove "value" prop if we are using formik.getFieldProps